plugins/adobeair/plugin.js

```diff
@@ -151,11 +151,12 @@
 					var panel = ui._.panel._.panel,
 						holder;
 
-					( function() {
+					( function addListeners() {
```

Comment:
Same here, this function waits for panel to be loaded, so something like `waitForPanel`, `waitForPanelToBeLoaded` could be used. Ofc, if panel is loaded the `convertInlineHandlers()` function is called which adds/manipulate listeners, but this is already different function.